### PR TITLE
PPCDEV-10038 update documentation regarding navigation timing page pe…

### PR DESCRIPTION
**Pull request by wronan**
…rformance metrics

### CODE REVIEW

File: analytics/js_tracking_api.rst [hunk 1]
```diff
@@ -859,3 +859,32 @@ To disable tracking user anonymously (after visitor gave consent) use ``deanonym
 
     _paq.push(["deanonymizeUser"]);
 
+
+
+Gethering navigation timing page performance metrics
```
Comment (azarecki) [inline analytics/js_tracking_api.rst]: Please fix typo

File: analytics/js_tracking_api.rst [hunk 2]
```diff
@@ -859,3 +859,32 @@ To disable tracking user anonymously (after visitor gave consent) use ``deanonym
...
+Argument to this function represents data sampling percentage (with possible integer values between 0 and 100).
+
+Example of usage::
+
+    _paq.push(["setTimingDataSamplingOnPageLoad", 0]); // disables timing data collection
+    _paq.push(["setTimingDataSamplingOnPageLoad", 30]); // enables 30% data sampling (only around 30% of all tracked actions will collect timing data if possible)
+    _paq.push(["setTimingDataSamplingOnPageLoad", 100]); // enables 100% data sampling (which means that all tracked actions will collect timing data if possible)
+
+IMPORTANT: by default this feature is disabled and no timing data is collected at any chance
+IMPORTANT: if enabled, timing data is collected only when page view lasted longer than the time it takse the page to load no partial information is stored, all metrics or nothing
```
Comment (azarecki) [inline analytics/js_tracking_api.rst]: There is no new line here, please add it

File: analytics/js_tracking_api.rst [hunk 3]
```diff
@@ -859,3 +859,32 @@ To disable tracking user anonymously (after visitor gave consent) use ``deanonym
...
+``````````````````````````
+To set up page performance metrics gathering use the ``setTimingDataSamplingOnPageLoad`` function::
+
+    _paq.push(["setTimingDataSamplingOnPageLoad", updateTimingDataOnPageLoadSampling]);
+
+.. describe:: updateTimingDataOnPageLoadSampling
+
+    **Required** ``integer`` Value between 1 and 100 describing the percentage for data sampling
+
+IT sets another request triggered onLoad, after trackPageView setting timing data.
```
Comment (azarecki) [inline analytics/js_tracking_api.rst]: `IT` is a typo or you wanted it in capital letters?

File: analytics/js_tracking_api.rst [hunk 4]
```diff
@@ -859,3 +859,32 @@ To disable tracking user anonymously (after visitor gave consent) use ``deanonym
 
     _paq.push(["deanonymizeUser"]);
 
+
+
+Gethering navigation timing page performance metrics
+``````````````````````````
+To set up page performance metrics gathering use the ``setTimingDataSamplingOnPageLoad`` function::
+
+    _paq.push(["setTimingDataSamplingOnPageLoad", updateTimingDataOnPageLoadSampling]);
```
Comment (azarecki) [inline analytics/js_tracking_api.rst]: Please add information when a user should run this `_paq.push`, before or after `trackPageView` etc.

File: analytics/js_tracking_api.rst [hunk 5]
```diff
@@ -859,3 +859,32 @@ To disable tracking user anonymously (after visitor gave consent) use ``deanonym
...
+
+Argument to this function represents data sampling percentage (with possible integer values between 0 and 100).
+
+Example of usage::
+
+    _paq.push(["setTimingDataSamplingOnPageLoad", 0]); // disables timing data collection
+    _paq.push(["setTimingDataSamplingOnPageLoad", 30]); // enables 30% data sampling (only around 30% of all tracked actions will collect timing data if possible)
+    _paq.push(["setTimingDataSamplingOnPageLoad", 100]); // enables 100% data sampling (which means that all tracked actions will collect timing data if possible)
+
+IMPORTANT: by default this feature is disabled and no timing data is collected at any chance
```
Comment (azarecki) [inline analytics/js_tracking_api.rst]: Please wait with this statement after we discuss it tomorrow within our team.

**Comment by azarecki** [comment]
I confirmed that we should merge it to branch `9.0`, please change :)

File: analytics/js_tracking_api.rst (hunk 2 above)
Comment (adaqus) [inline analytics/js_tracking_api.rst]: `takse` -> `takes`

**Pull request merged**